raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -232,7 +234,7 @@ scenario:
                 event_args: {closing_participant: 3}
 
             ## The MS reacts within the settle_timeout
-            - wait_blocks: 500
+            - wait_blocks: 40
```

Comment:
Shouldn't we try to somehow make the block waiting times variable depending on whatever the reveal/settle timeout of the contracts they run on are?

If not we would need to tweak the values each time.

Reply:
The SP has to support what you're suggesting based on the info returned by the channel endpoint. Or instead of waiting for blocks, we just wait for the channel status to become settled.